kaldb/src/test/java/com/slack/kaldb/testlib/MessageUtil.java

```diff
@@ -76,6 +76,10 @@ public static LogMessage makeMessageWithIndexAndTimestamp(
     Map<String, Object> fieldMap = new HashMap<>();
     fieldMap.put(LogMessage.ReservedField.TIMESTAMP.fieldName, timeStamp.toString());
     fieldMap.put(LogMessage.ReservedField.MESSAGE.fieldName, msgStr);
+    fieldMap.put(TEST_SOURCE_INT_PROPERTY, i);
```

Comment:
this did not fail any random tests?

Reply:
It did not - the other test helper methods already added these additional fields, but this one specifically did not. I went ahead and added these so I could more easily test the other aggregators (ie, avg).